source/device/cpu/cpu_device.c

```diff
@@ -50,7 +51,16 @@
 int init_cpu(struct device* device)
 {
     (void)device;
-    return register_all_cpu_ops();
+    int ret = unregister_all_cpu_ops();
+    if (0 != ret)
+    {
+        TLOG_ERR("Tengine: Unregister cpu devices failed: %d\n", ret);
+    }
+    else
+    {
+        release_cpu_node_ops_registry();
+    }
+    return ret;
```

Comment:
if does not register cpu op, then it's no need to unregister any op....

Reply:
oh, my mistake, I originally intended to add it to the next release_cpu function